docs/topics/request-response.rst

```diff
@@ -311,6 +311,7 @@ Those are:
 * ``ftp_password`` (See :setting:`FTP_PASSWORD` for more info)
 * :reqmeta:`referrer_policy`
 * :reqmeta:`max_retry_times`
+* :reqmeta:`allow_offsite_requests`
```

Comment:
Since this is a value for `Request.meta`, maybe it could be named simply `allow_offsite`?

Reply:
Agreed.